src/main/java/com/conveyal/gtfs/loader/PatternReconciliation.java

```diff
@@ -37,46 +51,91 @@ public enum PatternType {
         LOCATION, STOP
     }
 
+    /**
+     * Set required parameters and clear any previously held data to prevent it impacting current processing.
+     */
+    public static void initialize(Connection connection, String tablePrefix) throws SQLException {
```

Comment:
I would have written an instance constructor instead (with relevant fields and methods non-static)...